src/db_options.rs

```diff
@@ -177,6 +176,20 @@ impl Options {
         }
     }
 
+    /// If enable_pipelined_write is true, separate write thread queue is
+    /// maintained for WAL write and memtable write. A write thread first enter WAL
+    /// writer queue and then memtable writer queue. Pending thread on the WAL
+    /// writer queue thus only have to wait for previous writers to finish their
+    /// WAL writing but not the memtable writing. Enabling the feature may improve
+    /// write throughput and reduce latency of the prepare phase of two-phase
+    /// commit. db_bench benchmark shows 20% write throughput improvement
+    /// with concurrent writers and WAL enabled
```

Comment:
Add example please like this
```rust
    /// Default: `false`
    ///
    /// # Example
    ///
    /// ```
    /// use rocksdb::Options;
    ///
    /// let mut opts = Options::default();
    /// opts.enable_pipelined_write(true);
    /// ```
```